README.md

```diff
@@ -44,6 +44,7 @@ It doesn't require any modification to your application's code and let's you col
     ```ini
     extension=apm.so
     ```
+    Note: APM depends on JSON, so the apm.so extension must be loaded **after** JSON!
```

Comment:
Note that this is only true for Red Hat/Fedora/CentOS distributions as @remicollet pointed out. When loading .so, they are checking the fact that all symbols could be resolved, while normally it would crashes at runtime when calling an unknown symbol.

Note: if you use APM on such distribution I guess that you could use the RPM provided by @remicollet.